library/std/src/env.rs

```diff
@@ -350,18 +392,23 @@ impl Error for VarError {
 /// # Examples
 ///
 /// ```
+/// # #![allow(unused_unsafe)]
+///
 /// use std::env;
 ///
 /// let key = "KEY";
-/// env::set_var(key, "VALUE");
+/// unsafe {
+///     env::set_var(key, "VALUE");
+/// }
 /// assert_eq!(env::var(key), Ok("VALUE".to_string()));
 /// ```
+#[cfg(bootstrap)]
```

Comment:
The bootstrap-only version of the function never shows up in docs, so it doesn't need a doc comment (or only a dummy comment to satisfy a lint). That should avoid the duplication and make the diff easier to review.

Reply:
Done.